rosdep/python.yaml

```diff
@@ -1213,6 +1213,17 @@ python-crccheck-pip:
   ubuntu:
     pip:
       packages: [crccheck]
+python-crcmod:
+  debian: [python-crcmod]
+  fedora:
+    pip:
+      packages: [crcmod]
+  osx:
+    pip:
+      packages: [crcmod]
+  ubuntu:
+    '*': [python-crcmod]
```

Comment:
Since the python2 version does not look like it's going to go forward, instead of blacklisting 'focal' instead we should just explicitly list xenial and bionic and not list the other platforms.

Reply:
Done in a6c5635